src/components/utils/test/date_time_test.cc

```diff
@@ -108,11 +108,10 @@ TEST(DateTimeTest, CalculateTimeSpan) {
   // arrange
   const TimevalStruct time = date_time::DateTime::getCurrentTime();
 
-  const uint32_t sleep_time_mSec = 10;
+  const uint64_t sleep_time_mSec = 100;
   threads::sleep(sleep_time_mSec);
-
   // assert
-  ASSERT_GE(date_time::DateTime::calculateTimeSpan(time), sleep_time_mSec);
+  ASSERT_GE(date_time::DateTime::calculateTimeSpan(time) + 1, sleep_time_mSec);
```

Comment:
@orhan-mehmedov , @pestOO , The problem is how we get current time, on windows platform we call GetSystemTimeAsFileTime(). This method returns time in nanoseconds. When we convert nanoseconds to miliseconds, we just div it to 1000. 

For example: we call sleep(30) // wait 30 miliseconds
GetSystemTimeAsFileTime() - can return 29.9999997 / 1000 = 29

and test will fail.